.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -6,42 +6,10 @@ body:
       attributes:
           value: |
               Thanks for taking the time to fill out this bug report! If this is a security issue, please report it in HackerOne instead: https://hackerone.com/wordpress
-
-    - type: checkboxes
-      attributes:
-          label: Is there an existing issue for this?
-          description: Please check if the bug has already been reported by searching https://github.com/WordPress/gutenberg/issues.
-          options:
-              - label: I have searched the existing issues
-                required: true
-
-    - type: checkboxes
-      attributes:
-          label: Have you tried deactivating all plugins except Gutenberg?
-          description: Please make sure you have ruled out plugin conflicts before reporting.
-          options:
-              - label: I have tested with all plugins deactivated.
-                required: true
-
-    - type: checkboxes
-      attributes:
-          label: Have you tried replicating the bug using a default theme e.g. Twenty Twenty?
-          description: Please make sure you have confirmed it's not a theme specific problem.
-          options:
-              - label: I have tested with a default theme.
-                required: true
-
-    - type: textarea
-      attributes:
-          label: Description
-          description: Please write a brief description of the bug.
-      validations:
-          required: true
-
     - type: textarea
       attributes:
-          label: Step-by-step reproduction instructions
-          description: Please list the steps needed to reproduce the bug.
+          label: Description ands step-by-step reproduction instructions
+          description: Please write a brief description of the bug and the steps needed to reproduce it.
           placeholder: |
```

Comment:
Since this is now for both a description and steps needed to reproduce, I'd include something about the description in the placeholder section.

Reply:
Any copy suggestions? I feel that adding "When I do '...', it happens '...' is way too similar to the Expected vs Current behavior.